pkg/githubclient/client_repository.go

```diff
@@ -247,15 +248,34 @@ func (c *Client) getGithubChecks(ctx context.Context, repository *github.Reposit
 	var checks []string
 	for _, combinedStatus := range allCombinedStatus {
 		for _, status := range combinedStatus.Statuses {
-			checks = append(checks, *status.Context)
+			if filterCheck(status.GetContext()) {
+				checks = append(checks, status.GetContext())
+			}
 		}
 	}
 
 	c.logger.Debugf("found %d commit statuses for ref %q:\n%v", len(checks), ref, checks)
+	for id, check := range checks {
+		c.logger.Debugf(" - checks[%d] = %q", id, check)
+	}
```

Comment:
Now we are printing `checks` twice.

Let's make a choice, either keep it in the first (257) line, or keep the loop but not both.
I am fine either way.